appveyor.yml

```diff
@@ -0,0 +1,40 @@
+# What Python version is installed where:
+# https://www.appveyor.com/docs/build-environment/#python
+
+environment:
+  matrix:
+    - PYTHON: "C:\\Python35"
+      TOX_ENV: "py35"
+
+    - PYTHON: "C:\\Python36"
+      TOX_ENV: "py36"
+
```

Comment:
@AlejandroFernandezLuces Most of the project (PyVista, Pyansys, Numpy...) are not supporting version prior to 3.7.
I think here we can get rid of those versions (3.5 and 3.6).
```suggestion
```